addons/sys_core/XEH_postInit.sqf

```diff
@@ -93,6 +93,9 @@ if (!GVAR(aceLoaded)) then {
     [] call EFUNC(sys_components,toggleAntennaDir)
 }, [DIK_UPARROW, [false, true, true]]] call CBA_fnc_addKeybind;
 
+// Setup chat commands
+["acre-enable", { [false] call FUNC(setPaused); }] call CBA_fnc_registerChatCommand;
+["acre-disable", { [true] call FUNC(setPaused); }] call CBA_fnc_registerChatCommand;
```

Comment:
Two options
1. Move to documentation, so people can copy/paste it into their missions. Can also put it into one of the examples shipped with ACRE2.
2. Change into CBA Setting. (probably preferred)

I don't have preference of either, I think CBA Setting is easier as people can set it in-game, but CBA doesn't have a public setter (though many people use `CBA_settings_fnc_set`). Benefit over just documented API is also that it can be enabled in missions not created by you.